.github/workflows/ci.yml

```diff
@@ -12,8 +12,12 @@ jobs:
   test:
     timeout-minutes: 30
     strategy:
+      fail-fast: false
       matrix:
-        node-version: [ 18.x, 20.x, 21.x, 22.x ]
+        api_version:
```

Comment:
Should we add checks for the standard/experimental to ci-win.yml too?

Reply:
Good thought... I pretty much copied Gabriel's CI file. I'll comment on his PR to verify, since mine would end up being overwritten when i rebase to master after his is merged.